here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/AbstractTask.java

```diff
@@ -624,23 +621,51 @@ public final boolean removeListener(@NotNull Consumer<@NotNull RESULT> listener)
     }
   }
   /**
-   * Increments the value of authorUsage in the hashmap.
+   * Increments the value of authorUsage in the hashmap atomically.
    *
-   * @param author The author.
+   * @param principal The author.
    */
-  private static void incrementAuthorUsage(String author) {
-    authorUsageMap.computeIfAbsent(author, key -> new AtomicInteger()).incrementAndGet();
+  private static void incrementPrincipalUsage(String principal) {
```

Comment:
If this function is not used anymore, can we remove at the end?

Reply:
Fixed.